frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -98,38 +99,59 @@ function createFormGroup(controlId, formLabel, inputType, placeholder, ref) {
   }
 
   return (
-    <Form.Group controlId={controlId}>
+    <>
       <Form.Label>{formLabel}</Form.Label>
-        {formControl}
-    </Form.Group>
+      <Form.Group controlId={controlId}>
+          {formControl}
+        {/* Temporary instructions until fix Issue #52 */}
+      </Form.Group>
+    </>
   )
 }
 
 /**
- * Component corresponding to add trips modal.
+ * Component corresponding to the save trips modal.
+ *
+ * This component "acts" as a parent of the (non-existant) AddTripModal and
+ * EditTripModal components. The only differences in the implementation between
+ * the two fake components are dervied from the props `title`, `tripid`, and
+ * `placeholderObj` (see below).
  *
  * @param {Object} props These are the props for this component:
  * - db: Firestore database instance.
  * - show: Boolean that determines if the add trips modal should be displayed.
  * - handleClose: The function that handles closing the add trips modal.
```

Comment:
curious- What are the potential functions handleClose can be?

Reply:
Only `hideSaveTripModal`